packages/react/src/components/variants-provider.component.tsx

```diff
@@ -56,7 +56,7 @@ const variantsScript = (variantsString: string, contentId: string) =>
   }
   var cookieName = 'builder.tests.${contentId}';
   var variantInCookie = getCookie(cookieName);
-  var availableIDs = variants.map(function(vr) { return vr.id });
+  var availableIDs = variants.map(function(vr) { return vr.id }).concat("${contentId}");
```

Comment:
nit: looks like we use single quotes in most places, is prettier not formatting this?
```suggestion
  var availableIDs = variants.map(function(vr) { return vr.id }).concat('${contentId}');
```

Reply:
This is an interpolated string that we print in ssr